config.go

```diff
@@ -17,6 +18,8 @@ type Config struct {
 	Endpoint        string            `yaml:"endpoint"`
 	StrictBroadcast bool              `yaml:"strict_broadcast"`
 	ProxySetHeader  map[string]string `yaml:"proxy_set_header"`
+	SendTimeout     time.Duration     `yaml:"send_timeout"`
```

Comment:
Must be set default value(ex. `"1s"`) for backward compatibility.
time.Duration's zero value is 0s.